WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListActivity.java

```diff
@@ -63,17 +64,7 @@ protected void onNewIntent(Intent intent) {
     }
 
     private void handleIntent(Intent intent) {
-        mIsPage = intent.getBooleanExtra(EXTRA_VIEW_PAGES, false);
```

Comment:
My understanding is that `pageHasChanged` was always false since `mIsPage` was initialized just before the comparison.